api/web3server.go

```diff
@@ -673,13 +673,13 @@ func (svr *Web3Server) getTransactionReceipt(in interface{}) (interface{}, error
 		for _, tpc := range v.Topics {
 			topics = append(topics, "0x"+hex.EncodeToString(tpc[:]))
 		}
-		addr := ""
-		if contractAddr != nil {
-			addr = *contractAddr
+		addr, err := ioAddrToEthAddr(v.Address)
+		if err != nil {
+			return nil, err
 		}
 		log := logsObject{
 			BlockHash:        "0x" + blkHash,
-			TransactionHash:  actHashStr,
+			TransactionHash:  "0x" + hex.EncodeToString(v.ActionHash[:]),
```

Comment:
this is not necessary? see below

Reply:
tx hash without `0x` should be added prefix